bin/changelog

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-restricted-globals, no-console */
+
+const path = require('path');
+const { promises: fs } = require('fs');
+const util = require('util');
+
+const chalk = require('chalk');
+const execa = require('execa');
+const inquirer = require('inquirer');
+const lodash = require('lodash');
+const glob = util.promisify(require('glob'));
+const moment = require('moment');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+
+const hotConfig = require('../hot.config');
+
+const isTTY = process.stdout.isTTY;
+
+const changelogMarkDownMark = '<!-- UNVERSIONED -->';
+const changelogPath = path.join(__dirname, '../CHANGELOG.md');
+const changelogsDirectoryName = '.changelogs';
+
+const changelogEntryTypes = ['changed', 'removed', 'added', 'deprecated', 'fixed'];
```

Comment:
Since we've agreed that we will add `**breaking change**` prefix. I would want to back out from the order change I've proposed previously. It does not add any value, just breaks the backward compatibility with our changelog so far. WDYT @jansiegel @scarletfog?

Reply:
Yup, let's keep the identifier for a breaking change too.

Reply:
Restored the order in https://github.com/handsontable/handsontable/pull/7448/commits/94ca812caa307d235f1768ef5bd90b4c025fe4ef, breaking change changes described in https://github.com/handsontable/handsontable/pull/7448#discussion_r555636517.